### PR TITLE
[risk=no][RW-2261][RW-2492] change all frontend 'clone' language to 'duplicate'

### PR DESCRIPTION
**Pull request by jacmrob**

### CODE REVIEW

File: ui/src/app/app-routing.module.ts [hunk 1]
```diff
@@ -102,7 +102,7 @@ const routes: Routes = [
                 component: WorkspaceEditComponent,
                 data: {
                   title: 'Clone Workspace',
```
Comment (jmthibault79) [inline ui/src/app/app-routing.module.ts]: Update title

Comment (jacmrob) [reply ui/src/app/app-routing.module.ts]: Good catch

Comment (jmthibault79) [reply ui/src/app/app-routing.module.ts]: Q: should we also update the `clone` path?

**Comment by jacmrob** [comment]
This also includes changing API endpoints and routes, however the backend methods the correspond to are currently still using "clone"

**Pull request merged**